python_modules/libraries/dagster-airlift/dagster_airlift/in_airflow/gql_queries.py

```diff
@@ -106,6 +106,10 @@
 		... on Run {
 			id
 			status
+      tags {
+        key
+        value
+      }
 			__typename
```

Comment:
is this just github formatting being weird?

Reply:
I think this was previously weirdly formatted